set.mm

```diff
@@ -114740,6 +114791,16 @@ this axiom (with the defined operation in place of ` x. ` ) follows as a
     IZUKBDHIHIZUDUEUJUMJZUFUBUCUEUPUBUCUEKUIULDHABCLMNOUDUHEFZUGUJUNJZABPUQUEUF
     URUHCDQRSUBUEUCUFUMUOJZUBUEGUKEFZUCUFGUSACPUTUCUFUSUKBDQRSTUA $.
 
+  $( Rearrangement of 4 factors: swap the right factors in the factors of a
+     product of two products.  (Contributed by AV, 4-Mar-2023.) $)
+  mul4r $p |- ( ( ( A e. CC /\ B e. CC ) /\ ( C e. CC /\ D e. CC ) ) ->
```

Comment:
I suppose the ideal naming convention would somehow have a `23` in it for what is now `mul4` and a `24` for this one. Like `4mul23` and `4mul24` perhaps? Possibly more than you want to tackle right now, though (especially since at least to me it isn't clear what the best choice is).

Reply:
> I suppose the ideal naming convention would somehow have a `23` in it for what is now `mul4` and a `24` for this one. Like `4mul23` and `4mul24` perhaps? Possibly more than you want to tackle right now, though (especially since at least to me it isn't clear what the best choice is).

Yes, you are right: "r" means "reversed" in this case, and this suffix is also mentioned in the conventions:

> r : theorem with something reversed (e.g., a biconditional)